### DISCUSSION TITLE
New Awesome Papers, Books, and Libraries?

### DISCUSSION THREAD

**Discussion by o-laurent** [giannifranchi/awesome-uncertainty-deeplearning#1] [Ideas]
Here is the place for you to propose papers or everything else related to uncertainty in deep learning!

**Comment by wangyucheng1234** [comment]
Thank you for maintaining this excellent repository! It has been a valuable resource for researchers working on uncertainty quantification problems in ML like me.

I would like to kindly suggest adding our recent paper to the collection:

1. [Hashing with Uncertainty Quantification via Sampling-based Hypothesis Testing](https://openreview.net/pdf?id=cc4v6v310f) (TMLR):
This paper develops a probabilistic hashing model(ProbHash). Sampling-based hypothesis testing is then derived for hashing with uncertainty quantification(HashUQ) in ProbHash to improve the granularity of hashing-based retrieval by prioritizing the data with confident hash-codes. 

2. [Uncertainty-aware Unsupervised Video Hashing](https://proceedings.mlr.press/v206/wang23i.html) (AISTATS23): 
This paper proposes to formulate video hashing as learning a discrete variational auto-encoder with the factorized Bernoulli latent distribution, which enables uncertainty-aware video hashing by predicting the probability distribution of video hash code-words, thus providing reliable uncertainty quantification.

Thank you for considering my suggestion, and please let me know if you need any additional information!

Best regards, 
Yucheng Wang

**Reply by xuanlongORZ** [reply]
Thanks for your appreciation and contribution; I suggest placing the two papers in Bayesian-methods section if you agree. You can also send a pull request and do it by yourself.

Best,
Xuanlong

**Reply by wangyucheng1234** [reply]
Thank you for your reply!

I agree that these two papers could potentially fit well under the Bayesian-methods section. Please let me know if there’s any input you’d like from me. 

Best regards, 
Yucheng
